.github/workflows/stg-workflow.yml

```
@@ -0,0 +1,85 @@
+    branches:
+      - stg
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  main-api:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)